src/config.ts

```diff
@@ -192,6 +192,54 @@ const deleteProfile = (profileName: string): void => {
   writeConfig(AWS_CREDENTIALS_PATH, awsCredentials);
 };
 
+const getAssumeRoleProfiles = (parentProfile?: string): AssumeRoleProfileConfiguration[] => {
+  const awsConfig = getConfig(AWS_CONFIG_PATH);
+
+  const profiles: AssumeRoleProfileConfiguration[] = [];
+
+  for (const profile in awsConfig) {
```

Comment:
I think this should be a `for...of` loop, not a `for...in` loop.

Reply:
Since this is an object with a key we need to access, I don't _think_ that works

Reply:
In that case you would probably be better off using `for (const profile of Object.keys(awsConfig))`